js/privatebin.js

```diff
@@ -4360,7 +4370,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
          */
         me.getExpiration = function()
         {
-            return pasteExpiration;
+            return Model.getExpirationDefault() || pasteExpiration;
```

Comment:
The reason is that bootstrap3 does not use standard select-elements, but some lists with links and when clicked an event handler updates the hidden select and the stored `pasteExpiration` var. But page and bootstrap5 have a regular select without JS event, so `pasteExpiration` never gets updated. Model.getExpirationDefault() reads the val() from the select. And we fall back to the stored var for bootstrap3.

Yes, this means that selecting the expiration didn't work with the page template anymore. We didn't touch this bit in 7 years, so it was broken for some time. This has convinced me that no one still uses the page template, or they'd reported this issue.

Reply:
> This has convinced me that no one still uses the page template, or they'd reported this issue.

So we can remove it?

Reply:
We can, but that is definitely another PR and out of scope for this one. Also not a prio to me, as it is not the one that gets reports on shipping vulnerable JS libs and it causes no maintenance burden at this time.

What I absolutely want to avoid is not shipping at least two templates, because that will inevitably lead to us adding template specific code to JS and then templating becomes much more difficult, if not impossible.